src/intro_gui.cpp

```diff
@@ -259,17 +260,55 @@ struct IntroGameAnimator {
 };
 
 
+Point PositionSelectGameWindow(Window *w)
+{
+	Point p;
+	const int edge_offset = ScaleGUITrad(20);
+	const int width = w->width;
+	const int height = w->height;
+
+	switch (w->window_number) {
+		case 0:
```

Comment:
Could there be an enum for these window numbers? (Honest question, I know they're not widgets but maybe we could treat them similarly.)